src/sonic-ctrmgrd/tests/ctrmgrd_test.py

```diff
@@ -0,0 +1,493 @@
+}
+
+
+labels_test_data = {
+    0: {
+        common_test.DESCR: "simple update",
+        common_test.ARGS: "ctrmgrd",
+        common_test.CONNECTED: True,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.UPD: {
+            common_test.STATE_DB_NO: {
+                "xyz": {
+                    "xxx": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.POST: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.KUBE_CMD: {
+            common_test.KUBE_WR: {
+                "foo": "bar",
+                "hello": "world"
+            }
+        }
+    },
+    1: {
+        common_test.DESCR: "simple update -  not connected",
+        common_test.ARGS: "ctrmgrd",
+        common_test.CONNECTED: False,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.POST: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        }
+    },
+    2: {
+        common_test.DESCR: "Emulate write failure",
+        common_test.ARGS: "ctrmgrd",
+        common_test.CONNECTED: True,
+        common_test.KUBE_RETURN: 1,
+        common_test.PRE: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.POST: {
+            common_test.STATE_DB_NO: {
+                common_test.KUBE_LABEL_TABLE: {
+                    "SET": {
+                        "foo": "bar",
+                        "hello": "world"
+                    }
+                }
+            }
+        },
+        common_test.KUBE_CMD: {
+            common_test.KUBE_WR: {
+                "foo": "bar",
+                "hello": "world"
+            }
+        }
+    }
+}
+
+
+class TestContainerStartup(object):
+
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
```

Comment:
`setup_class()` is a unittest concept. For pytest, fixtures should be used instead. Since this is basically a no-op, suggest removing.